images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -10,7 +10,15 @@ ENV NGINX_RUN_USER www-data
 ENV NGINX_DOCUMENT_ROOT /home/site/wwwroot
```

Comment:
If NGINX only installed in PHP-FPM images? For my knowledge, could you please clarify the difference between PHP-FPM and the standard PHP?

Reply:
Yes, NGINX only exists in PHP FPM runtime images. The main difference is that standard PHP images use Apache server while FPM images use NGINX as a load balancing server.

Reply:
Thanks for clarifying